mscore/musescore.cpp

```diff
@@ -7390,12 +7390,12 @@ MuseScoreApplication* MuseScoreApplication::initApplication(int& argc, char** ar
       const char* appName;
       const char* appName2;
       if (MuseScore::unstable()) {
-            appName2 = "mscore-dev3";
-            appName  = "MuseScore3Development";
+            appName2 = "mscore-dev4";
+            appName  = "MuseScore4Development";
             }
       else {
-            appName2 = "mscore3";
-            appName  = "MuseScore3";
+            appName2 = "mscore4";
+            appName  = "MuseScore4";
```

Comment:
While we're at them, isn't it better to put spaces between words? `MuseScore 4 Dev` / `MuseScore 4`? Are there downsides for using spaces in this particular scenario?

Reply:
I'd rather not do that.

Reply:
But they're also used for some dialogues in the app, and it's not consistent to see "MuseScore 4" on the main window but "MuseScore4" for others.

Reply:
Where exactly? Anyhow, I see no compelling reason to change this.

Reply:
As far as I can tell that `appName` is only used for path names (and for telemetry)